setup/setup_fedora.sh

```diff
@@ -0,0 +1,182 @@
+# Install everything else
+sudo dnf -y install ack
+sudo dnf -y install git git-extras hub 
+sudo dnf -y install htop ngrep nmap
+sudo dnf -y install autojump # A smart command to change directory (https://github.com/wting/autojump)
+pip install legit            # http://www.git-legit.org/
+legit install		     # Enables the extra git commands
+sudo dnf -y install zopfli   # Data compression software (https://github.com/google/zopfli)
+sudo dnf -y install cowsay
+# pup not available for Linux (Fedora)
+sudo dnf -y install sshrc    # https://github.com/Russell91/sshrc
+pip install stormssh         # Linux alternative for storm https://github.com/emre/storm
+pip install --user howdoi
+sudo dnf -y install httpie   # https://github.com/jakubroztocil/httpie
+sudo dnf -y install jq       # https://stedolan.github.io/jq/
+sudo dnf -y install elixir
+sudo dnf -y install ShellCheck
+sudo dnf -y install teamviewer
+
+###############################################################################
+#                        Install utilities                                    #
+###############################################################################
+
+sudo dnf -y install firefox
+#Evernote,skitch,dash not available
+sudo dnf -y install https://prerelease.keybase.io/keybase_amd64.rpm
+#android-file-transfer-linux http://whoozle.github.io/android-file-transfer-linux/
+#sudo dnf -y install chromium # open source alternative of Google Chrome
+#sudo dnf -y install hexchat  # open source alternative of limechat
+sudo dnf -y install dropbox
+sudo dnf -y install vlc
+# nvalt alternative nvpy for notes https://github.com/cpbotha/nvpy
+
+# Postgres Database
+sudo dnf -y install postgresql-server
+sudo dnf -y install postgresql
+sudo dnf -y install pgadmin3
+sudo dnf -y install rubygems
+sudo gem install pg
+
+# PG tools 
+# osgeo needs epel
+sudo dnf -y install postgis
+
+################################################################################
+#                           Datat Stores                                       #
+################################################################################
+
+sudo dnf -y install mysql
+sudo dnf -y install mongodb
+sudo dnf -y install redis
+sudo dnf -y install elasticsearch
+
+################################################################################
+#                           Dev tools                                          #
+################################################################################
+sudo dnf -y install VirtualBox
+sudo dnf -y install vagrant
+sudo dnf -y install python-postman
+sudo dnf -y install weechat
+sudo dnf -y install tmux
+pip install --user cookiecutter
+
+#Frontend stuff
+sudo dnf -y install npm
+npm i -g postcss-cli
+npm i -g autoprefixer
+
+#Custom stuff
+#Telegram
+sudo dnf -y copr enable rommon/telegram
+sudo dnf -y install telegram-desktop
+sudo dnf copr disable rommon/telegram
+
+# Slack needs to be unpacked manually due to version name in download url
```

Comment:
I think we can include a particular version here and update it later, when needed. Does it have some sort of problem with it?

Reply:
No nothing. Will update.